bin/manage.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+consulCommand() {
+    consul-cli --quiet --consul="${CONSUL}:8500" $*
```

Comment:
What's this getting us that we don't already have from the `consul` binary in the container?